e2e/test/iothub/messaging/MessageReceiveE2ETests.cs

```diff
@@ -247,7 +247,7 @@ private async Task DoNotReceiveMessagesSentBeforeSubscriptionAsync(TestDeviceTyp
             {
                 await testDeviceCallbackHandler.WaitForReceiveMessageCallbackAsync(cts.Token).ConfigureAwait(false);
             };
-            receiveMessageOverCallback.Should().Throw<OperationCanceledException>();
+            await receiveMessageOverCallback.Should().ThrowAsync<OperationCanceledException>();
```

Comment:
There are tighter type validations done by fluent assertions now, so all these updates are to resolve compile errors.